package-lock.json

```diff
@@ -0,0 +1,3265 @@
+{
```

Comment:
I don't think we need package lock. We need bun lock

Reply:
Ok, got it. I wasn't sure if you had wanted both lockfiles for cross-compatibility. I'll remove package-lock.json then since we don't need it (and it's more confusing to have both lockfiles). 

The bun lock file is already added [here ](https://github.com/aalmacin/social-media-optimizer/blob/5b6df4fb86ac5c27181c46a79ec7d50f2e631120/bun.lockb).